test/router_data_SUITE.erl

```diff
@@ -454,8 +454,8 @@ data_test_3(Config) ->
             <<"hotspot">> => #{
                 <<"id">> => erlang:list_to_binary(libp2p_crypto:bin_to_b58(PubKeyBin)),
                 <<"name">> => erlang:list_to_binary(HotspotName),
-                %% RSSI act as power here we are and based on UPLINK RSSI < -80 we should power downlink at 27
-                <<"rssi">> => 27,
+                %% RSSI act as power here we are and based on UPLINK RSSI < -80 we should power downlink at 20
+                <<"rssi">> => 20,
```

Comment:
This comment is a bit confusing to me when read with the one above.
Most of the RSSI's in the tests are changing to 30.
These two are the outliers, and I don't understand from the comments why they're both changing to 20 under opposite circumstances.

```
Above comment from L315
%% RSSI act as power here we are and based on UPLINK RSSI > -80 we should power downlink at 20

vs. this comment L457
%% RSSI act as power here we are and based on UPLINK RSSI < -80 we should power downlink at 20
```

Reply:
Lets pair code review this tomorrow.

Reply:
We found this to be a misunderstanding of the way gateway regions and device regions are used to compute the downlink power.